crates/services/bls_aggregation/src/bls_agg.rs

```diff
@@ -636,9 +636,15 @@ impl<A: AvsRegistryService + Send + Sync + Clone + 'static> BlsAggregatorService
             return Err(SignatureVerificationError::OperatorPublicKeyNotFound);
         };
 
+        let message = signed_task_response_digest
+            .task_response_digest
+            .as_slice()
+            .try_into()
```

Comment:
does this adds padding automatically if the digest is not 32 bytes?

Reply:
no, it just parses it when it is exactly 32 bytes long